source/usage-examples/findOne.txt

```diff
@@ -11,6 +11,7 @@ Find a Document
    :values: tutorial
 
 .. meta:: 
+   :description: Learn how to find one document in your MongoDB collection by using the Node.js driver.
```

Comment:
Suggestion:

Similar to the suggestions provided for source/usage-examples/find.txt, I think one of these would keep them similar to each other:

```suggestion
   :description: Learn how to retrieve one document from MongoDB by using the Node.js driver.
```

```suggestion
   :description: Follow a usage example of retrieving one document from MongoDB by using the Node.js driver.
```